PSOC6: rebuild libwiced_drivers.a against mbed-os-5.11.4

vmedcy opened the pull request:
### Description

The WICED precompiled libraries submitted with mbed-os-5.11.4
were compiled against mbed-os master branch at 57d8915a.

At that moment, the master branch contained commits targeting
mbed-os-5.12, this prevented the libraries from functioning correctly
when manually picked to mbed-os-5.11 release branch.

The libraries are now compiled against ecb3c8c.

### Pull request type

    [ ] Fix
    [ ] Refactor
    [X] Target update
    [ ] Functionality change
    [ ] Docs update
    [ ] Test update
    [ ] Breaking change

### Reviewers

@SeppoTakalo

ciarmcom commented:
@vmedcy, thank you for your changes.
@SeppoTakalo @ARMmbed/mbed-os-maintainers please review.

ciarmcom commented:
@vmedcy, thank you for your changes.
@SeppoTakalo @ARMmbed/mbed-os-maintainers please review.

0xc0170 commented:
> to merge 1 commit into ARMmbed:master

This should be targeting 5.11 branch rather?

vmedcy committed:
PSOC6: rebuild libwiced_drivers.a against mbed-os-5.11.4

The WICED precompiled libraries submitted with mbed-os-5.11.4
were compiled against mbed-os master branch at #57d8915a:
https://github.com/ARMmbed/mbed-os/tree/57d8915a

At that moment, the master branch contained commits targeting
mbed-os-5.12, this prevented the libraries from functioning correctly
when manually picked to mbed-os-5.11 release branch.

The libraries are now compiled against #ecb3c8c:
https://github.com/ARMmbed/mbed-os/tree/ecb3c8c

vmedcy commented:
@0xc0170: yes, target is mbed-os-5.11, not master.
Changed the PR target and rebased the commit.

cmonr commented:
CI job restarted: `jenkins-ci/greentea-test`

A couple of the machines failed to initialize properly and the job was simply idling.
